From 6b1e4fc4a8d41e2813531bd40af685e63d367581 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Tue, 20 Dec 2016 15:35:12 +0900
Subject: [PATCH 1426/2066] Revert "arm64: dts: r8a7796: Connect Ethernet AVB
 to IPMMU"

This reverts commit 1995c505654fe338cfa54c01c515e450bf5a48fe.
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index 6059ad3..fbfb80e 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -476,7 +476,6 @@
 			phy-mode = "rgmii-id";
 			#address-cells = <1>;
 			#size-cells = <0>;
-			iommus = <&ipmmu_ds0 16>;
 		};
 
 		i2c0: i2c@e6500000 {
-- 
1.7.5.4

